### DISCUSSION TITLE
Scroll

### DISCUSSION THREAD

**Discussion by parimaleshwaran** [tailwindlabs/tailwindcss#3316] [Help]
how to customize in smooth scroll

**Comment by radohingis** [comment]
if you need to customize scroll-bar then you can use this https://www.npmjs.com/package/tailwind-scrollbar
it is the simplest but efficient plugin that adds classes for customizing scrollbars in tailwind.

**Comment by Williamssonn** [comment]
Get the current top location using self. pageYOffset.
Get the position of element till where you want to scroll to: element. offsetTop.
Do a for loop to reach there, which will be quite fast or use a timer to do smooth scroll till that position using window. scrollTo.
[
targetpayandbenefits](https://www.targetpayandbenefits.biz/)
